packages/libraries/core/package.json

```diff
@@ -42,7 +42,6 @@
     "@fluentui-react-native/pressable": "0.11.6",
     "@fluentui-react-native/radio-group": "0.20.8",
     "@fluentui-react-native/separator": "0.17.6",
-    "@fluentui-react-native/tabs": "0.14.7",
```

Comment:
(non-blocking) j.w. should TabList be in core?

Reply:
Yes it should be. Added.